File: edu-platform-master/edu-platform-master/src/main/java/edu/platform/domains/learner/model/response/LearnerResponse.java
package edu.platform.domains.learner.model.response;

import edu.platform.domains.learner.constant.LearnerStatus;
import lombok.Data;

@Data
public class LearnerResponse {
    private Long id;
    private String familyName;
    private String firstName;
    private String email;
    private Integer yearOfBirth;
    private Long provinceId;
    private Long districtId;
    private Long wardId;
    private Boolean onlineCourseInterest;
    private Boolean inPersonCourseInterest;
    private LearnerStatus status;
    private Boolean isChildAccount;
}

File: edu-platform.web-master/edu-platform.web-master/src/services/learner/response/LearnerResponse.ts
import {BaseResponse} from "../../BaseResponse";

export interface Learner {
  id: number;
  familyName: string;
  firstName: string;
  email: string;
  yearOfBirth: number;
  provinceId: number;
  districtId: number;
  wardId: number;
  onlineCourseInterest: boolean;
  inPersonCourseInterest: boolean;
  status: string;
  isChildAccount: boolean;
}

export interface LearnerResponse extends BaseResponse {
  data: Learner
}